### PR TITLE
added requested fields

### PR DESCRIPTION
**Pull request by kunal15595**

### CODE REVIEW

File: object-models/common-models.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ KEY | TYPE | NULL/EMPTY | DESCRIPTION
 --- | --- | --- | ---
 code | string | no | Code name for the city. Example: `NEW_YORK`, `DUBAI`
 name | string | no | Display name of the city
+image | [`image`](common-models.md#image) | no | The primary image for the city.
```
Comment (vjsingh-lite) [inline object-models/common-models.md]: I think we created the image object for scenarios when we need to represent the image with other properties too. Like caption etc. Question: Should we use just `imageUrl` over here or the object?

**Review by vjsingh-lite — CHANGES_REQUESTED** [review]
Question.

File: object-models/common-models.md (hunk 1 above)
Comment (kunal15595) [reply object-models/common-models.md]: 1. we didn't have that requirement till now.
2. an object will be forward compatible and image url wont, works if new attributes need to be added or not.

**Review by vjsingh-lite — DISMISSED** [review]

File: object-models/common-models.md (hunk 1 above)
Comment (alpeshvas) [reply object-models/common-models.md]: Let's also add `image` field in Response example of CityAPI [here](https://github.com/headout/api-docs/blob/master/apis/city.md)

Comment (alpeshvas) [reply object-models/common-models.md]: Let's also add `image` field in Example response of CityAPI [here](https://github.com/headout/api-docs/blob/master/apis/city.md)

**Review by alpeshvas — CHANGES_REQUESTED** [review]

**Pull request merged**